app/actions/action_dropNewPin.js

```diff
@@ -51,15 +53,9 @@ export default function getLocationToSave(location, current, pinTitle) {
   }
   return (dispatch) => {
       if(!location) {
-        navigator.geolocation.getCurrentPosition(
-          (position) => {
-             getLocationHelper(position.coords, "My Current Location", dispatch);
-          },
-          (error) => {
-            console.error(error);
-          },
-          {enableHighAccuracy: true, timeout: 20000, maximumAge: 1000}
-        );
+        geoAction.getCurrent((loc)=>{
+          getLocationHelper(position.coords, "My Current Location", dispatch);
+        });
```

Comment:
You're referencing a variable `loc` in the anonymous function definition but don't seem to be making use of it here.